### ISSUE TITLE
Unable to host any game

### ISSUE THREAD

**Issue by Saracenus** [octgn/OCTGN#806]
Whenever I try and host a game on the server (or hosting on my computer) the hosted game briefly appears and then disappears off the Custom Games tab and I have to shut down to get out of it. (hosting on my computer gets a "There was a problem, please try again" message). I have cut and pasted part of the log file that seems to be the section dealing with a hosting failure.

I can join a game no problem.

ASUS Laptop:
- AMD Bronze Dual Core E450 1.65 GHz
- 4 GB RAM
- Windows 7 64bit
- Atheros AR9285 Wireless Card
- Realtek PCIe GBE Ethernet Port
- Norton Security Suite

Home Network (not working):
- Comcast internet
- Connecting via WiFi (no Ethernet option available at my location right now) using 802.11n
- Technicolor TC8305C Modem

Mom's Network (not working):
- Comcast internet
- Connecting via CAT5 Ethernet cable directly into modem (also tried via 802.11n).
- Technicolor TC8305C Modem

Guardian Games Network (working):
- Comcast internet
- Connecting via WiFi using 802.11g
- Different modem.

Red Castle Games (working):
- Provider Unknown
- Connecting via WiFi using 802.11n
- Modem/WiFi Router Unknown

This problem didn't start until I updated to 3.1 (I updated from my 3.0 install, but there was a 3.1 test install in parallel). Prior to this I had very stable connections via 3.0 and the test version of 3.1 worked perfectly. 

Troubleshooting steps taken:
- Uninstalled and res-installed.
- Re-uninstalled and re-installed but deleted the OCTGN folders for a clean install.
- Installed on a different computer (same issue on the new machine).
- I have had one instance where I was successful, but then it went right back to not working. This happened when I installed on the other computer (which failed to host) and then logged back in on my own computer. I repeated this and have not been able to host again.
- Turned off firewall (no joy)
- Different location (no joy)
- Test with Ethernet hardwired connect @ different location (no joy).
- Created new account and logged in with it (no joy).
- Found that Windows Firewall was turned on alongside Norton. Turned off Windows Firewall. (no joy).

Future steps
- I have not changed or messed with my firewall and NAT router ports, but given direction on which one I might need to open up to OCTGN I am willing to try it.

Previous problem self hosting turned out to be a separate issue that was solved with the 3.1.24.108 update.
